.github/workflows/push.yaml

```diff
@@ -145,7 +145,7 @@ jobs:
           cargo check -p but-workspace --all-targets
           cargo check -p but-github --all-targets
           cargo check -p but-api --all-targets
-          cargo check -p but --all-targets --no-default-features
+          cargo check -p but --all-targets
```

Comment:
The idea here was to have a 'modern' version to work towards, adding more and more commands as we go in a conscious effort.
Removing CI for this just means we can't build `modern` right now, and it *should* always be possible to make it compile by excluding non-compiling or legacy parts with `[cfg(feature = "legacy")]`.

I would try to get it back unless you want to use a different strategy to eventually get there. Without compiler help, I wouldn't know how to do that though 😅.

Reply:
Yes please, if there is a way to make it lets do that. The thing is, now the CLI has the "fetch" that is performed a little bit like the git GC - if certain conditions are met (i.e. time passed), running a command will do a fetch. 
Ideally it would be nice to update the fetch functionality to be non-legacy, rather than trying to cut it out